test/link-style-autolink-only.md

```diff
@@ -0,0 +1,20 @@
+# Autolink Link Style
+
+Text [url](https://example.com) text {MD054}
+Text ![url](https://example.com) text {MD054}
+Text [url] text {MD054}
+Text ![url] text {MD054}
+Text [text][url] text {MD054}
+Text ![text][url] text {MD054}
+Text <https://example.com> text
+Text [url][] text {MD054}
+
+[url]: https://example.com
```

Comment:
Once you make the edits to this file, please replicate that structure to each of the others - thanks!